pyproject.toml

```diff
@@ -63,6 +63,6 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project.scripts]
-pubtools-sign-msg-clear-sign = "pubtools.sign.signers.msgsigner:msg_clear_sign_main"
-pubtools-sign-msg-container-sign = "pubtools.sign.signers.msgsigner:msg_container_sign_main"
+pubtools-sign-msg-clear-sign = "pubtools.sign.signers.msgsigner:msg_clear_sign"
```

Comment:
The _main functions still exist, what is their purpose?

Reply:
no, main functions are needed I just forgot to revert it here